dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -219,11 +226,24 @@ def prepare_airflow_packages(
     sys.exit(result_command.returncode)
 
 
+def provider_documentation_summary(documentation: str, message_type: MessageType, packages: list[str]):
+    if packages:
+        get_console().print(f"{documentation}: {len(packages)}\n")
+        get_console().print(f"[{message_type.value}]{' '.join(packages)}")
+        get_console().print()
+
+
 @release_management.command(
     name="prepare-provider-documentation",
     help="Prepare CHANGELOG, README and COMMITS information for providers.",
 )
-@option_debug_release_management
+@click.option(
+    "--skip-git-fetch",
+    is_flag=True,
+    help="Skips removal and recreation of `apache-https-rof-providers` remote in git. By default, the "
```

Comment:
```suggestion
    help="Skips removal and recreation of `apache-https-for-providers` remote in git. By default, the "
```

Reply:
Yeah, It's massive, but I was not able to do it gradually :(. I am doing "command-by-command" on that one - so this one has some remnants that are removed only in the follow-up one (#35617  - and only there you see some of the code finally removed) 

Really sorry for that, but I found no other way to do it :( (so overall those 2 PRS combined actually remove(!) 300 lines of code - this one's +1300 is just an intermediate step). 

It's mostly really moving code from one place to the other while cleaning necessary bits to make the moved code fit in the new place.

Reply:
Having done way (too) many massive PRs, I understand :) I'll continue the review